### ISSUE TITLE
Anaconda Navigator - Start up error

### ISSUE THREAD

**Issue by shankarsubra** [ContinuumIO/anaconda-issues#7982]
Navigator Error
An unexpected error occurred on Navigator start-up

Report
Please report this issue in the anaconda issue tracker

Main Error
'root_prefix'
Traceback
Traceback (most recent call last):
  File "C:\Users\Shankar Subra\Anaconda3\lib\site-packages\anaconda_navigator\exceptions.py", line 75, in exception_handler
    return_value = func(*args, **kwargs)
  File "C:\Users\Shankar Subra\Anaconda3\lib\site-packages\anaconda_navigator\app\start.py", line 120, in start_app
    window = run_app(splash)
  File "C:\Users\Shankar Subra\Anaconda3\lib\site-packages\anaconda_navigator\app\start.py", line 63, in run_app
    window = MainWindow(splash=splash)
  File "C:\Users\Shankar Subra\Anaconda3\lib\site-packages\anaconda_navigator\widgets\main_window.py", line 161, in __init__
    self.api = AnacondaAPI()
  File "C:\Users\Shankar Subra\Anaconda3\lib\site-packages\anaconda_navigator\api\anaconda_api.py", line 1258, in AnacondaAPI
    ANACONDA_API = _AnacondaAPI()
  File "C:\Users\Shankar Subra\Anaconda3\lib\site-packages\anaconda_navigator\api\anaconda_api.py", line 70, in __init__
    self._conda_api = CondaAPI()
  File "C:\Users\Shankar Subra\Anaconda3\lib\site-packages\anaconda_navigator\api\conda_api.py", line 1654, in CondaAPI
    CONDA_API = _CondaAPI()
  File "C:\Users\Shankar Subra\Anaconda3\lib\site-packages\anaconda_navigator\api\conda_api.py", line 345, in __init__
    self.set_conda_prefix()
  File "C:\Users\Shankar Subra\Anaconda3\lib\site-packages\anaconda_navigator\api\conda_api.py", line 494, in set_conda_prefix
    self.ROOT_PREFIX = info['root_prefix']
KeyError: 'root_prefix'

**Comment by shankarsubra** [comment]
Navigator is very slow while starting up. In this instance is threw up this error

**Comment by alinnachen** [comment]
same problem here. I just updated Anaconda. When I launch the Anaconda Navigator after the update it gives this error message.

**Comment by alinnachen** [comment]
Navigator Error

An unexpected error occurred on Navigator start-up
Report

Please report this issue in the anaconda issue tracker
Main Error

byte indices must be integers or slices, not str

Traceback

Traceback (most recent call last):
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\anaconda_navigator\exceptions.py", line 75, in exception_handler
    return_value = func(*args, **kwargs)
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\anaconda_navigator\app\start.py", line 115, in start_app
    window = run_app(splash)
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\anaconda_navigator\app\start.py", line 58, in run_app
    window = MainWindow(splash=splash)
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\anaconda_navigator\widgets\main_window.py", line 160, in __init__
    self.api = AnacondaAPI()
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\anaconda_navigator\api\anaconda_api.py", line 1205, in AnacondaAPI
    ANACONDA_API = _AnacondaAPI()
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\anaconda_navigator\api\anaconda_api.py", line 65, in __init__
    self._conda_api = CondaAPI()
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\anaconda_navigator\api\conda_api.py", line 1622, in CondaAPI
    CONDA_API = _CondaAPI()
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\anaconda_navigator\api\conda_api.py", line 340, in __init__
    self.set_conda_prefix()
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\anaconda_navigator\api\conda_api.py", line 489, in set_conda_prefix
    self.ROOT_PREFIX = info['root_prefix']
TypeError: byte indices must be integers or slices, not str

**Comment by goanpeca** [comment]
Closing as duplicate of #1837.

---

Please remember to update to the latest version of Navigator to include
the latest fixes.

Open a terminal (on Linux or Mac) or the Anaconda Command Prompt (on windows)
and type:

```
$ conda update conda
$ conda update anaconda-navigator
$ conda update navigator-updater
```

---

**See issue #1837 for more information on how to fix this issue.**

**Comment by peeterx32** [comment]
I too faced the same issue earlier today. I tried using conda commands but I kept on getting an error 
                `ConfigurationLoadError: Unable to load configration file.`
                `reason: Invalid yaml at position 0 `

I tried deleting the `.condarc` folder and everything works great. No more `Navigator Error` on starting up Anaconda.

Try it out guys and who knows it might work, if the commands; 
`$ conda update conda
 $ conda update anaconda-navigator
 $ conda update navigator-updater`

fail to.

Cheers!
